### PR TITLE
Add mackerel plugin proc fd

### PR DESCRIPTION
**Pull request by taku-k**
This plugin monitor opening file descriptor.

### CODE REVIEW

File: mackerel-plugin-proc-fd/proc_fd.go [hunk 1]
```diff
@@ -0,0 +1,131 @@
...
+		[]string{"ps", "aux"},
+		[]string{"grep", p.Process},
+		[]string{"grep", "-v", "grep"},
+		[]string{"grep", "-v", "mackerel-plugin-proc-fd"},
+		[]string{"awk", "{print $2}"},
+	)
+	if err != nil {
+		// No matching with p.Process invokes this case
+		logger.Errorf("No matching with process")
+		return nil, errors.New("No matching with process")
```
Comment (yuuki) [inline mackerel-plugin-proc-fd/proc_fd.go]: Just `return nil, err`  better?

File: mackerel-plugin-proc-fd/proc_fd.go [hunk 2]
```diff
@@ -0,0 +1,131 @@
...
+	for _, pid := range strings.Split(strings.TrimSpace(string(out)), "\n") {
+		out, err = pipeline.Output(
+			[]string{"ls", "-l", fmt.Sprintf("/proc/%s/fd", pid)},
+			[]string{"grep", "-v", "total"},
+			[]string{"wc", "-l"},
+		)
+		if err != nil {
+			// The process with pid terminates"
+			logger.Errorf("The process terminates")
+			return nil, errors.New("The process terminates")
```
Comment (yuuki) [inline mackerel-plugin-proc-fd/proc_fd.go]: Ditto.

**Comment by yuuki** [comment]
Would you be able to provide a test ?

**Comment by taku-k** [comment]
If #215 is merged, this CI test will be passed

**Pull request merged**